src/test/kotlin/uk/gov/justice/digital/hmpps/locationsinsideprison/resource/SyncAndMigrateResourceIntTest.kt

```diff
@@ -67,10 +67,20 @@ class SyncAndMigrateResourceIntTest : SqsIntegrationTestBase() {
   lateinit var nonRes: NonResidentialLocation
   lateinit var locationHistory: LocationHistory
 
-  fun migrateRequestGeneration(deactivationReason: NomisDeactivatedReason): NomisMigrateLocationRequest {
+  enum class DeactivatedReasonOther {
```

Comment:
Did you need to refined the existing NomisDeactivatedReason enum?

Reply:
right i have simplified it. thanks